Marlin/src/lcd/extui/lib/mks_ui/draw_ready_print.cpp

```diff
@@ -138,20 +138,20 @@ void lv_draw_ready_print() {
 
     e1 = lv_label_create_empty(scr);
     lv_obj_set_pos(e1, 20, 20);
-    sprintf_P(buf, PSTR("e1:  %d"), (int)thermalManager.degHotend(0));
+    sprintf_P(buf, PSTR("e1:  %d"), thermalManager.degHotend(0));
```

Comment:
here too... etc you got it

Reply:
In fact, when using format strings, using the casts is the safer way to keep things working.